src/simsopt/field/magneticfieldclasses.py

```diff
@@ -517,7 +517,7 @@ class InterpolatedField(sopp.InterpolatedField, MagneticField):
     This resulting interpolant can then be evaluated very quickly.
     """
 
-    def __init__(self, field, degree, rrange, phirange, zrange, extrapolate=True, nfp=1, stellsym=False):
+    def __init__(self, field, degree, rrange, phirange, zrange, extrapolate=True, nfp=1, stellsym=False, skip=None):
```

Comment:
How about adding `skip` to the docstring that follows.

Reply:
good point, done